assets/panel/panel.js

```diff
@@ -50,6 +50,11 @@ DebuggerPanel.prototype = {
             .setNodeFront(front, { reason: "debugger" });
 
           return Promise.all([onNodeFrontSet, onInspectorUpdated]);
+        }.bind(this),
+        openConsoleAndEvaluate: async function(input) {
+          await this.toolbox.selectTool("webconsole");
+          const jsterm = this.toolbox.getPanel("webconsole").hud.jsterm;
+          jsterm.execute(input);
```

Comment:
that looks reasonable to me. At some point, I think we're going to move `execute` outside of jsterm to the panel, so we can avoid retrieving the jsterm reference.